### PR TITLE
Use new Docker images

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
We have begun to cleanup our Docker images and this makes it so that we use the new images.

The way we start Docker has been cleaned up. All containers now should use the `start-docker.sh` provided in the Docker image. Other than that, there are no real changes in this PR.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/test/e2e/api/cluster_rbac_test.go [hunk 1]
```diff
@@ -153,20 +161,16 @@ func getErrorResponse(err error) string {
 	return string(rawData)
 }
 
-func createProjectWithCluster(t *testing.T, testClient *utils.TestClient, dc, credential, version, location string, replicas int32) (*v1.Project, *v1.Cluster) {
+func createProjectWithCluster(t *testing.T, testClient *utils.TestClient, dc, credential, version, location string, replicas int32) (*v1.Project, *v1.Cluster, error) {
```
Comment (zreigz) [inline pkg/test/e2e/api/cluster_rbac_test.go:164]: :+1:

**Comment by xrstf** [comment]
The changes of this PR will probably be rolled up into another one. This can be closed for now.

**Pull request closed without merging**